nistats/first_level_model.py

```diff
@@ -20,6 +20,7 @@
 
 import numpy as np
 import pandas as pd
+import nibabel as nib
```

Comment:
Rather than this import, could you directly use Nifti1Image imported at the line below ?